examples/complete/README.md

```diff
@@ -1,9 +1,9 @@
 # Complete example with Encryption and CBR rules
 
-An end-to-end example that adds encryption to the [default example](../default/README.md). This example uses the IBM Cloud terraform provider to:
+An end-to-end example that adds encryption to the [basic example](../basic/README.md). This example uses the IBM Cloud terraform provider to:
```

Comment:
why are we referencing the basic example here? I would just remove that first sentence.